example/basic/basic.cxx

```diff
@@ -83,7 +83,9 @@ t8_basic_refine_test (t8_eclass_t eclass)
   t8_forest_t         forest_adapt;
```

Comment:
I dont think these changes come from you, do they?
I will ignore them for now.

Reply:
They are. I initially made them for a better understanding of this example and how the functions work. But they do not have something to do with subelements.